source/2015-11-27-colorize-svgs-with-sass.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+title: Colorize SVGs with SASS
+date: 2015-11-27 16:05 UTC
+author: bro
+tags: 
+---
+
+As you might know, it is possible to style an SVG image via CSS if you
+inline it. The problem, though, is that you cannot do this when the SVG
```

Comment:
According to the [Dictionary of Modern American Philosophers, Volume 1] (https://books.google.ch/books?id=Ijpj1tB3Qr0C&pg=PA585&lpg=PA585&dq=%22the+problem,+though%22+dictionary&source=bl&ots=8CeRbPuvEA&sig=RAWrohtp36QaBnUkmTcKYXq8FsE&hl=en&sa=X&redir_esc=y#v=onepage&q=%22the%20problem%2C%20though%22%20dictionary&f=false) and [other sources](http://www.linguee.com/english-german/search?source=auto&query=the+problem+though) they are correct :wink: . I'll keep them.